app/views/users/index.html.erb

```diff
@@ -1 +1,6 @@
-<h1>This is the index page where all users of the app will arrive</h1>
+<section class='all-users'>
+  <h1 class='shadow'>ALL OUR BLOGERS</h1>
+  <% @users.each do |user| %>
+    <%= render partial: 'user_box', locals: {user:user} %>
```

Comment:
- [ ] [OPTIONAL] You did a great job rendering a partial here and in other parts of the code-base 🥇

Reply:
Thank you!